ocf_datapipes/select/__init__.py

```diff
@@ -1,15 +1,24 @@
 """Selection datapipes"""
+from .assign_daynight_status import AssignDayNightStatusIterDataPipe as AssignDayNightStatus
```

Comment:
Could we move this to `.transforms.xarray` instead? It fits more under there along with the other metadata-adding transforms.